service/src/main/kotlin/fi/espoo/evaka/attendance/ChildAttendanceController.kt

```diff
@@ -148,7 +148,7 @@ class ChildAttendanceController(
             if (tx.getChildCurrentDayAttendance(childId, unitId) != null)
                 throw Conflict("Cannot arrive, already arrived today")
 
-            tx.deleteCurrentDayAbsences(childId)
+            tx.deleteAbsences(childId)
```

Comment:
nitpick: this sounds like all absences from that child will be nuked --> I would remove the default parameter from the date to make it more obvious what this actually does